### DISCUSSION TITLE
Web Scraping Basics

### DISCUSSION THREAD

**Discussion by AriZoneVibes** [justfoolingaround/animdl#243] [General]
@gamechanger2030 You can post your questions here and we can give you advice. 

Is there something in particular on the web that you are interested in scraping? Could use as a project to start learning.

**Comment by gamechanger2030** [comment]
umm, i am interested in regex for now, feels cool to use symbols to do stuffs
as for web scraping, i and manage basics with requests,api and by reading source code of websites but i don't know how to handle javascript loaded objects, requests can't pull it off
I also tried selenium, it can do mostly but i guess you know its load on system, so don't know how to handle javascripts, should i learn javascript or there are other ways?

**Comment by Shemusty** [comment]
I am conducting a research on evaluation of web scraping tools. i am finding it difficult to get dataset on scraping tools.

**Comment by Charlotte-br560** [comment]
Hey @AriZoneVibes, thanks for initiating this! @gamechanger2030, a great place to start learning web scraping is by picking a website you're interested in. It could be anything from news articles to product listings. And as @AriZoneVibes mentioned, [Crawlbase](https://crawlbase.com/) is a fantastic tool for beginners. Give it a try and feel free to ask any questions along the way!
